Sources/App/routes.swift

```diff
@@ -8,6 +8,10 @@ public func routes(
     gitHubEventsService: GitHubEventsService
 ) throws {
 
+    router.get("/") { request -> String in
```

Comment:
maybe better to do it `/status`?

Reply:
Initially I was using that route but maybe using the direct URL for the bot is easier to remember? 🤷‍♂️

Reply:
true